examples/basic_setup/README.MD

```diff
@@ -0,0 +1,7 @@
+## Setup
+
+This example assumes you have geckodriver or chromedriver listening at port 4444.
```

Comment:
Could you include the command-lines for running those tools on that port here? Would probably be useful to someone wanting to try it themselves.